test/cmd/idle.sh

```diff
@@ -16,21 +16,23 @@ prev_scale_annotation='idling.alpha.openshift.io/previous-scale'
 idled_at_template="{{index .metadata.annotations \"${idled_at_annotation}\"}}"
 unidle_target_template="{{index .metadata.annotations \"${unidle_target_annotation}\"}}"
 prev_scale_template="{{index .metadata.annotations \"${prev_scale_annotation}\"}}"
+dc_name=""
 
 setup_idling_resources() {
     os::cmd::expect_success 'oc delete all --all'
 
     # set up resources for the idle command
-    os::cmd::expect_success 'oc create -f test/extended/testdata/idling-echo-server.yaml'
-    os::cmd::expect_success 'oc describe deploymentconfigs idling-echo'
+    os::cmd::expect_success 'oc create -f test/testdata/idling-svc-route.yaml'
+    dc_name=$(basename $(oc create -f test/testdata/idling-dc.yaml -o name))  # `basename type/name` --> name
```

Comment:
I'm not particularly happy with this. In a follow-up we should either:
 - hard-code it
 - pass it in through a template
 - grab it using `jsonpath` from the metadata